### ISSUE TITLE
Editor display booleans should be display flags

### ISSUE THREAD

**Issue by gunchleoc** [widelands/widelands#4335]
These variables in editorinteractive should be display flags for consistency:

```
bool draw_resources_ = true;
bool draw_immovables_ = true;
bool draw_bobs_ = true;
bool draw_grid_ = true;
```

c.f. https://github.com/widelands/widelands/issues/3265#issuecomment-536178710

**Comment by akupar** [comment]
This is fixed in #5220

**Comment by tothxa** [comment]
Closed via #5220
